run/README.namelist

```diff
@@ -254,17 +254,21 @@ Namelist variables specifically for the WPS input for real:
                                                   = 2: it uses an alternative way (less biased 
                                                   when compared against input data) to compute height in program 
                                                   real and pressure in model.
+ wif_input_opt                       = 0        ! = 1: option to process the Water Ice Friendly Aerosol input from metgrid for use with mp_physics=28
+                                                  = 2: since V4.4, option to use black carbon aerosol category with mp_physics=28, as well as its radiative effect. Must include
+                                                       file QNWFA_QNIFA_QNBCA_SIGMA_MONTHLY.dat during WPS 
+ num_wif_levels                      = 27       ! number of levels in the Thompson Water Ice Friendly aerosols (mp_physic=28) 
```

Comment:
@kkeene44 The registry value for this namelist is now 30.

Reply:
@weiwangncar Okay, I changed it to 30.